OWNERS

```diff
@@ -1,2 +1 @@
-sara@pantheon.io
-ronan@pantheon.io
+cms_ecosystem@pantheon.io
```

Comment:
This should be a GitHub group; c.f. https://github.com/pantheon-systems/autopilot-services/blob/master/CODEOWNERS#L4

We'll have to create a group for CMS Ecosystems first, if one doesn't already exist.

Reply:
like this one: https://github.com/orgs/pantheon-systems/teams/cms-ecosystem